src/event.rs

```diff
@@ -67,17 +71,12 @@ pub enum Event<T: 'static> {
     /// See [`ApplicationHandler::window_event`] for details.
     ///
     /// [`ApplicationHandler::window_event`]: crate::application::ApplicationHandler::window_event
-    WindowEvent { window_id: WindowId, event: WindowEvent },
+    Window { window_id: WindowId, event: WindowEvent },
```

Comment:
I'm also in favor of moving this change into a different PR.

Reply:
It's a private type now, so rename doesn't really matter at this point? Also, I think some clippy option was throwing a warning because of that, since private `Event` type has variants ending with `Event`.

Reply:
Why does it matter if its private or not in regards of doing it in a separate PR?

Reply:
I've renamed it back and silenced clippy.